### DISCUSSION TITLE
[Problem] VSCodium not opening unless I use the console/terminal.

### DISCUSSION THREAD

**Discussion by Gui2Barbieri** [clearfraction/distribution#41] [Q&A] [answered]
Hi, I having a problem with latest Clear Linux Wayland GNOME version (40390) and Clear Fraction.
I installed the VSCodium bundle, but it doesnt't quite work. I mean, it does work if I launch it via console/terminal using "codium", but the desktop entry in GNOME doesn't work, it just hangs up loading but never launches.
What can I do to fix this? Thanks!

**Comment by paulcarroty** [comment, marked as answer]
Still actual. Solution: create the alias in `~/.bashrc`: `alias codium='codium --ozone-platform=wayland'`.
